src/jenkins.py

```diff
@@ -3,11 +3,14 @@
 
 """Functions to operate Jenkins."""
 
+# pylint:disable=too-many-lines
+
```

Comment:
Refactoring of the jenkins module should be done in another PR, currently out of scope.

Reply:
This seems to be something that is up for debate - usually I try to fix these kind of linting issues the first time they come up. If I find myself spending too much time on it or it increases the complexity of the PR by a lot then it can be left to a future PR, otherwise it can be reasonable to do it in the same PR.

The solution here could be just to split this file into two or three separate files with logically grouped functions.

Systematically leaving these kind of issues in a PR can lead to a lot of follow up PRs that we may forget to get to. Most other linting issues introduced in a PR are usually fixed within a PR

Reply:
Another way to address this is to do it in a follow up PR and associate it with the same Jira ticket/ issue so that we don't call the feature done until the linting issues introduced by it have been resolved

Reply:
@jdkandersson This can now be removed since the later PR #60 has already been merged and has reduced the LOC significantly in `jenkins.py`. :) Updating now!